src/more-infos/more-info-lock.html

```diff
@@ -0,0 +1,16 @@
+<link rel='import' href='../../bower_components/polymer/polymer.html'>
+
+<link rel='import' href='../../bower_components/paper-button/paper-button.html'>
+<link rel='import' href='../../bower_components/paper-input/paper-input.html'>
+
+<dom-module id='more-info-lock'>
+  <template>
```

Comment:
None of the buttons should be showed when there is no code needed to unlock a lock.